### DISCUSSION TITLE
New idea for a thing!!

### DISCUSSION THREAD

**Discussion by devinhalladay** [devinhalladay/groves#20] [Q&A] [answered]
Here is an idea :)

**Comment by devinhalladay** [comment]
Here is a way I imagine implementing this idea

It's good, right? Let's mark it as the answer

**Reply by winnieyoe** [reply]
super cool best idea

**Comment by winnieyoe** [comment, marked as answer]
A very important idea

**Reply by devinhalladay** [reply]
> A very important idea

This is a great idea, I will mark it as the answer!

**Comment by devinhalladay** [comment]
Let's see what happens when I paste the reference link https://github.com/devinhalladay/groves/discussions/20#discussioncomment-169686
